### ISSUE TITLE
Spectrogram?

### ISSUE THREAD

**Issue by turian** [keunwoochoi/kapre#103]
I have an older version of Kapre that has time_frequency.Spectrogram, which is trainable.

However, the new version of Kapre doesn't have Spectrogram anymore. Why?

**Comment by keunwoochoi** [comment]
With the new kapre, it would be replaced with `kapre.STFT()` followed by `kapre.Magnitude()`. 

I changed the name because precisely, the implementation is different.

**Comment by vincenzodentamaro** [comment]
But what does the trainable means? I am confused.
If we are doing trainable STFT, means that kernels (feature maps) for such STFT are trained by gradient descent or constrained by the STFT formula?

**Comment by keunwoochoi** [comment]
The current STFT is not trainable so rest assured. 

The previous Spectrogram used conv layer-based DFT which could be set as trainable, which meant trainable kernels initialized to function as DFT.

**Comment by vincenzodentamaro** [comment]
Thank you for your answer.
I am using kapre.time_frequency.Spectrogram and Melspectogram.
What is the equivalent in this version? What should I do to compose both of them?

**Comment by keunwoochoi** [comment]
they are provided in composed layer. please check out the doc! https://kapre.readthedocs.io/

**Comment by vincenzodentamaro** [comment]
Yes indeed. Thanks

**Comment by vincenzodentamaro** [comment]
Just to know, intuitively a trainable melbank weight matrix should improve the quality of results because it is finetuned on the dataset. Am I wrong?

**Comment by bjarthur** [comment]
@vincenzodentamaro if you want a trainable front-end, see https://github.com/google-research/leaf-audio.  the related work section in the corresponding [paper](https://arxiv.org/abs/2101.08596) provides many other alternatives too.  and the [blog](https://ai.googleblog.com/2021/03/leaf-learnable-frontend-for-audio.html).
